stores/filestore.go

```diff
@@ -3401,6 +3576,7 @@ func (c *msgsCache) evict(now int64) {
 		// Bulk remove
 		c.seqMaps = make(map[uint64]*cachedMsg)
 		c.head, c.tail, c.tryEvict = nil, nil, 0
+		runtime.GC()
```

Comment:
What does this block?

Reply:
This should be fairly good these days with golang, not sure this is needed.

Reply:
I put that at one point because during some test bench, the mem usage was going quite high and so I used `-profile` and looked at memory in use and it was high. After adding this call, all mem usage for Unmarshal and add to cache disappeared, so that convinced me that I did not have a mem leak. So I will remove but the GC is not aggressive because that inuse_space stays for several minutes at the end of a test...

Reply:
ok just concerned this does a block here inline which we should avoid if we can.